specs/core/0_beacon-chain.md

```diff
@@ -1596,41 +1611,35 @@ _Note_: If there are skipped slots between a block and its parent block, run the
 
 ### Per-slot processing
 
-Below are the processing steps that happen at every slot.
-
-#### Slot
+Below are the processing steps that happen at every `slot > GENESIS_SLOT`.
```

Comment:
So first we do book-keeping for the previous slot? This smells a bit funny

Reply:
Agreed. Fixed!